***************
*** 1641,1645 ****
    { test ! -d "$ac_tmp" || rm -fr "$ac_tmp"; } && exit $exit_status
  ' 0
!   trap 'AS_EXIT([1])' 1 2 13 15
  }
  dnl The comment above AS_TMPDIR says at most 4 chars are allowed.
--- 1641,1645 ----
    { test ! -d "$ac_tmp" || rm -fr "$ac_tmp"; } && exit $exit_status
  ' 0
!   trap 'AS_EXIT([1])' HUP INT PIPE TERM
  }
  dnl The comment above AS_TMPDIR says at most 4 chars are allowed.
