diff --git a/configure b/configure
index 4fbcab8..89b9b86 100755
--- a/configure
+++ b/configure
@@ -1672,7 +1672,7 @@ ac_exit_trap ()
 }
 
 trap 'ac_exit_trap $?' 0
-for ac_signal in 1 2 13 15; do
+for ac_signal in HUP INT PIPE TERM; do
   trap 'ac_signal='$ac_signal'; as_fn_exit 1' $ac_signal
 done
 ac_signal=0
@@ -4542,7 +4542,7 @@ $debug ||
   : "${ac_tmp:=$tmp}"
   { test ! -d "$ac_tmp" || rm -fr "$ac_tmp"; } && exit $exit_status
 ' 0
-  trap 'as_fn_exit 1' 1 2 13 15
+  trap 'as_fn_exit 1' HUP INT PIPE TERM
 }
 # Create a (secure) tmp directory for tmp files.
 
