diff --git a/tests/local.at b/tests/local.at
index 95b5f15..518beae 100644
--- a/tests/local.at
+++ b/tests/local.at
@@ -366,6 +366,8 @@ m4_case([$4], [], [], [ignore], [],
         s/^\([^:]*:\) *\([0-9][0-9]*:\)[^:]*m4[-.ex0-9]*: /m4:\1\2 /
         s/: include: [cC]\(annot open\)/: c\1/
         s/: [cC]\(annot open \)[`'\'']*\([^'\'':]*\)'\''*:/: c\1'\''\2'\'':/
+	s/\(.*\)\( EDC5129I \)\(.*\)\.$/\1 \3/
+	s/\(.*\)\( EDC[0-9][0-9][0-9][0-9]I \)\(.*\)/\1 \3/
         s/^autom4te: [^ ]*m4[.ex]* /autom4te: m4 /
         s/^autom4te: error: [^ ]*m4[.ex]* /autom4te: error: m4 /
         s!^.*/\([^/][^/]*\)\.m4: *[0-9][0-9]*: *!\1.m4: !
