diff --git a/lib/autoconf/status.m4 b/lib/autoconf/status.m4
index ca8cb69..f4fb49d 100644
--- a/lib/autoconf/status.m4
+++ b/lib/autoconf/status.m4
@@ -1640,7 +1640,7 @@ $debug ||
   : "${ac_tmp:=$tmp}"
   { test ! -d "$ac_tmp" || rm -fr "$ac_tmp"; } && exit $exit_status
 ' 0
-  trap 'AS_EXIT([1])' 1 2 13 15
+  trap 'AS_EXIT([1])' HUP INT PIPE TERM
 }
 dnl The comment above AS_TMPDIR says at most 4 chars are allowed.
 AS_TMPDIR([conf], [.])
